### PR TITLE
contact form updated

### PR DESCRIPTION
**Pull request by Ankita297**
## What is the change?
contact form updated
## Related issue?
close: #838 

## How was it tested?
I have tested it in my local system .
## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.
WordNook - Google Chrome 12-05-2021 12_14_52
WordNook - Google Chrome 12-05-2021 13_04_42
WordNook - Google Chrome 12-05-2021 13_04_34
WordNook - Google Chrome 12-05-2021 13_04_42
WordNook - Google Chrome 12-05-2021 13_04_53

### CODE REVIEW

File: public/css/styles.css [hunk 1]
```diff
@@ -1124,3 +1124,84 @@ svg {
...
+	position: absolute;
+	top: 15px;
+	left: 0;
+	-webkit-transition: all 0.25s ease;
+	transition: all 0.25s ease;
+	pointer-events: none;
+}
+
+.styled-input.wide {
+	width: 650px;
```
Comment (Pranav016) [inline public/css/styles.css:1165]: I would suggest refrain from using `px` to set width, use `vw`

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Good work,
image

This blue bar at the top looks weird and you can remove it. Add a bit more border radius to the whole form area and PR will be good to merge 👍 

Also run `npm run lint` before your next commit.

File: public/css/styles.css (hunk 1 above)
Comment (Ankita297) [reply public/css/styles.css]: Sir i have done the changes where i can and i have tested it . and npm run lint is failing due to first commit . 
Sorry for that i will keep that in mind from next time .

**Comment by Ankita297** [comment]
Sir done the changes 

WordNook - Google Chrome 12-05-2021 14_23_32
WordNook - Google Chrome 12-05-2021 14_23_08

**Review by Pranav016 — APPROVED** [review]

**Comment by Ankita297** [comment]
Thankyou sir

File: views/navitems/contact.ejs [hunk 2]
```diff
@@ -11,21 +12,30 @@
...
         <div class="form-group">
-            <label>Message:</label>
-            <textarea class="form-control col-md-12 col-sm-12 col-xs-12" rows="6" name="message" placeholder="Your Message" value="<%= formData.message %>" required></textarea>
+            <div class="styled-input wide">
+              <label>Message:</label>
+              <textarea class="form-control col-md-12 col-sm-12 col-xs-12" rows="6" name="message" placeholder="Your Message" value="<%= formData.message %>" required></textarea>
+            </div>
         </div>
-        <button class="btn btn-primary block" type="submit" size="50dp">Send</button>
+        <button class="btn btn-primary  send-btn" type="submit">Send</button>
```
Comment (Pranav016) [inline views/navitems/contact.ejs:36]: remove the `btn` `btn-primary` class from this and reuse the button class there in the styles.css
and then add the extra css to this send-btn class. Add all the colors and hover again to this class doesn't make sense.
btn-primary should also be removed, no point of having it

File: views/navitems/contact.ejs [hunk 3]
```diff
@@ -11,21 +12,30 @@
...
+              <label>Subject:</label>
+              <input class="form-control col-md-12 col-sm-12 col-xs-12" type="text" name="subject" placeholder="Subject" autocomplete="off" value="<%= formData.name %>" required></input>
+           </div>
         </div>
         <div class="form-group">
-            <label>Message:</label>
-            <textarea class="form-control col-md-12 col-sm-12 col-xs-12" rows="6" name="message" placeholder="Your Message" value="<%= formData.message %>" required></textarea>
+            <div class="styled-input wide">
+              <label>Message:</label>
+              <textarea class="form-control col-md-12 col-sm-12 col-xs-12" rows="6" name="message" placeholder="Your Message" value="<%= formData.message %>" required></textarea>
```
Comment (Pranav016) [inline views/navitems/contact.ejs:33]: use col-12 class instead of all of these

Comment (Pranav016) [inline views/navitems/contact.ejs:33]: col-12

**Review by Pranav016 — COMMENTED** [review]
Make these changes as well to clean the code

File: views/navitems/contact.ejs (hunk 3 above)
Comment (Ankita297) [reply views/navitems/contact.ejs:33]: okay Sir

**Comment by Ankita297** [comment]
profile page enhanced by Ankita297 · Pull Request #883 · ALPHAVIO_WordNook - Google Chrome 15-05-2021 15_22_07
is this okay ?

**Comment by Pranav016** [comment]
Make PR or whatever changes I had requested, I'll review them then..

**Pull request merged**